README.md

```diff
@@ -109,7 +104,9 @@ See [FAQ.md](/FAQ.md).
 
 See [DEVELOPMENT.md](/DEVELOPMENT.md).
 
-## 🤝 Contributing [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+## 🤝 Contributing
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
```

Comment:
As we now have the badges in the start, it might be nice to move this "PRs welcome" also there.

Reply:
Good idea. Should we keep it here as well or just have it at the top?

Reply:
I think that only at the top is enough.

Reply:
And.. could you remove the following from the definition of this badge as it is having different style 😅 
```
?style=flat-square
```

Reply:
you have a keen eye. I didn't even notice that in the preview :D

Reply:
Thanks! Now it looks purrrfect 😸